modules/virt-configuring-vm-dpdk.adoc

```diff
@@ -0,0 +1,120 @@
+
+You can run Data Packet Development Kit (DPDK) workloads on virtual machines (VMs) to achieve lower latency and higher throughput for faster packet processing in the user space. DPDK uses the SR-IOV network for hardware-based I/O sharing.
+
+.Prerequisites
+* Your cluster is configured to run DPDK workloads.
+* You have created and configured the project in which the VM will run.
+
+.Procedure
+. Edit the `VirtualMachine` manifest to include information about the SR-IOV network interface, CPU topology, CRI-O annotations, and huge pages:
++
+.Example `VirtualMachine` manifest
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: rhel-dpdk-vm
+spec:
+  running: true
+  template:
+    metadata:
+      annotations:
+        cpu-load-balancing.crio.io: disable <1>
+        cpu-quota.crio.io: disable <2>
+        irq-load-balancing.crio.io: disable <3>
+    spec:
+      nodeSelector:
+        node-role.kubernetes.io/worker-dpdk: "" <4>
+      domain:
+        cpu:
+          sockets: 1 <5>
+          cores: 5 <6>
+          threads: 2
+          dedicatedCpuPlacement: true
+          isolateEmulatorThread: true
+        interfaces:
+          - masquerade: {}
+            name: default
+          - model: virtio
+            name: nic-east
+            pciAddress: '0000:07:00.0'
+            sriov: {}
+          networkInterfaceMultiqueue: true
+          rng: {}
+      memory:
+        hugepages:
+          pageSize: 1Gi <7>
+      resources:
+        requests:
+          memory: 8Gi
+      networks:
+        - name: default
+          pod: {}
+        - multus:
+            networkName: dpdk-net <8>
+          name: nic-east
+# ...
+----
+<1> This annotation specifies that load balancing is disabled for CPUs that are used by the container.
+<2> This annotation specifies that the CPU quota is disabled for CPUs that are used by the container.
+<3> This annotation specifies that Interrupt Request (IRQ) load balancing is disabled for CPUs that are used by the container.
+<4> The label that is used in the `MachineConfigPool` and `PerformanceProfile` manifests that were created when configuring the cluster for DPDK workloads.
+<5> The number of sockets inside the VM. This field must be set to `1` for the CPUs to be scheduled from the same Non-Uniform Memory Access (NUMA) node.
+<6> The number of cores inside the VM. This must be a value greater than or equal to `1`. In this example, the VM is scheduled with 5 hyper-threads or 10 CPUs.
+<7> The size of the huge pages. The possible values for x86-64 architecture are 1Gi and 2Mi. In this example, the request is for 8 huge pages of size 1Gi. 
+<8> The name of the SR-IOV `NetworkAttachmentDefinition` object.
+
+. Save and exit the editor.
+. Apply the `VirtualMachine` manifest:
++
+[source,terminal]
+----
+$ oc apply -f <file_name>.yaml
+----
+
+. Configure the guest operating system. The following example shows the configuration steps for RHEL 8 OS:
```

Comment:
Does this also work for sub versions of RHEL 8? That might be helpful to specify